src/xrpld/app/misc/NetworkOPs.cpp

```diff
@@ -1475,12 +1495,13 @@ NetworkOPsImp::apply(std::unique_lock<std::mutex>& batchLock)
                 auto const toSkip =
                     app_.getHashRouter().shouldRelay(e.transaction->getID());
 
-                if (toSkip)
+                if (auto const txn = *(e.transaction->getSTransaction());
```

Comment:
```suggestion
                if (auto const sttx = *(e.transaction->getSTransaction());
```

consider renaming `txn` since it conflicts with `tx`